C/CryptoMiniSat/build_tarballs.jl

```diff
@@ -6,30 +6,68 @@ const YGGDRASIL_DIR = "../.."
 include(joinpath(YGGDRASIL_DIR, "platforms", "mpi.jl"))
 
 name = "CryptoMiniSat"
-version = v"5.11.4"
+version = v"5.13.0"
```

Comment:
```suggestion
version = v"6.0.0"
cryptominisat_version = v"5.13.0"
```

So would this be reasonable as we can't guarantee backwards compatibility due to `ExecutableProduct("cryptominisat5_simple", :cryptominisat5_simple)` missing now?